cypress/e2e/tests/rbac/migrator.test.ts

```diff
@@ -39,6 +39,7 @@ describe(["@tier3", "@rhsso"], "Migrator RBAC operations", () => {
     );
 
     before("Creating RBAC users, adding roles for them", () => {
+        cy.clearLocalStorage();
```

Comment:
Thanks @ibragins . I'm curious to know if there is any particular data in local storage that needs to be cleared. Also, any reason why cy.clearLocalStorage()  is being used only in this file and not in architect.test.ts ?